client/app/bundles/KudosApp/components/Kudo.jsx

```diff
@@ -61,21 +82,49 @@ export default class Kudo extends React.Component {
   }
 
   render() {
+
+    const Edit = () => <button
+      type='button'
+      className="btn update-button edit-button"
```

Comment:
Where is the styling for `update-button` defined? I feel like it should be near the styling for `edit-button`. (Also since `edit-button` includes `btn` styling, why does this also need to have the `btn` classname?)